packages/infra/heroku/init.sh

```diff
@@ -59,21 +59,32 @@ heroku pg:credentials:create nft-storage-prod-0 --name=nft_storage --app=nft-sto
 heroku pg:psql nft-storage-staging-0 --app=nft-storage-staging < grant-postgrest.sql
 heroku pg:psql nft-storage-prod-0 --app=nft-storage-prod < grant-postgrest.sql
 
-# Configure the DB_URI and JWT_SECRET for PostgREST
-heroku config:set DB_URI=$(heroku config:get DATABASE_URL --app=nft-storage-staging) --app=nft-storage-pgrest-staging
-heroku config:set DB_URI=$(heroku config:get DATABASE_URL --app=nft-storage-prod) --app=nft-storage-pgrest-prod
-# Obtain secret from 1password vault!
-heroku config:set JWT_SECRET="supersecret" --app=nft-storage-pgrest-staging
-heroku config:set JWT_SECRET="supersecret" --app=nft-storage-pgrest-prod
+# Attach databases to apps
+heroku addons:attach nft-storage-prod-0 --app=nft-storage-prod
+heroku addons:attach nft-storage-replica-0 --app=nft-storage-prod
+heroku addons:attach nft-storage-staging-0 --app=nft-storage-staging
+
+# PGREST config vars
```

Comment:
Does this not work with the config files anymore?

Reply:
it still works with config files, but i feel the `heroku git:remote` step is super error prone and we cant have different configs for prod and staging in the same repo. So i think its best to just use the config vars until we find a better solutions.